[DX-1258] - Adding missed changelog items into the Dashboard RN (#4407)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md

</li>
<li>
<details>

> <summary>Eliminated the necessity of mock response descriptions</summary>

Reviewer: @andyo-tyk @Eopayemi 

Does the change-log item refer to:
- Removed mandatory (required) status of mock response description OR
- Removing validation formatting etc. of mock response descriptions

Reviewer 2: ```suggestion
<summary>Removed the need for a Description to be provided in the OpenAPI schema when autogenerating a Tyk OAS mock response</summary>
```

> Removed strict validation for the description field in mock responses when utilising Tyk OAS, ensuring flexibility for response status codes and headers.

Reviewer 2: ```suggestion
Relaxed the strict validation for mock response so that the `Description` field is now optional for `response`, `responses` and `schema` within the OpenAPI description. Automatically configuring mock responses when using [Tyk OAS APIs]({{< ref "product-stack/tyk-gateway/middleware/mock-response-openapi" >}}) is now even easier.
```

</details>
</li>
<li>
<details>

> <summary>Organisation Name Update via cURL Enhancement</summary>

Reviewer: ```suggestion
Updated OAS documentation for updating the organisation name
```

Reviewer 2: Do we usually tag docs improvements in release notes? Not sure if this one is required if it's just a docs change (even if it is resolving a user reported issue).

> Improved the documentation to explain the usage of *PUT /admin/organisations/{ORG_ID}*

Reviewer 2: I don't think this is required (per comment above).
